mobile/library/swift/EngineBuilder.swift

```diff
@@ -171,6 +172,20 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Specify whether to enable DNS cache.
+  ///
+  /// Note that DNS cache requires an addition of a key value store named
+  /// 'reserved.platform_store'.
```

Comment:
Seems odd to have the user provide a "reserved" KV store.

Reply:
agree but that requirement was not introduced in this PR. IMO the API for adding key value stores is too flexible - which I think is something that we already discussed in the past. Will need to close it down a bit in the future.